src/SwedbankPay.Sdk/PaymentOrders/PaymentOrderRequestContainer.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace SwedbankPay.Sdk.PaymentOrders
+{
+    public class PaymentOrderRequestContainer
```

Comment:
What is the purpose of this class?

Reply:
To make the serialization match the json request